Source/WebCore/Modules/mediastream/gstreamer/GStreamerMediaEndpoint.cpp

```diff
@@ -2278,7 +2278,17 @@ void GStreamerMediaEndpoint::onIceCandidate(guint sdpMLineIndex, gchararray cand
     if (candidateString.isEmpty())
         return;
 
-    callOnMainThread([protectedThis = Ref(*this), this, sdp = WTFMove(candidateString).isolatedCopy(), sdpMLineIndex]() mutable {
+    auto descriptions = descriptionsFromWebRTCBin(m_webrtcBin.get());
+
+    String mid;
+    GUniqueOutPtr<GstWebRTCSessionDescription> description;
+    g_object_get(m_webrtcBin.get(), "local-description", &description.outPtr(), nullptr);
+    if (description && sdpMLineIndex < gst_sdp_message_medias_len(description->sdp)) {
+        const auto media = gst_sdp_message_get_media(description->sdp, sdpMLineIndex);
+        mid = unsafeSpan(gst_sdp_media_get_attribute_val(media, "mid"));
+    }
+
+    callOnMainThread([protectedThis = Ref(*this), this, sdp = WTFMove(candidateString).isolatedCopy(), sdpMLineIndex, descriptions = WTFMove(descriptions), mid = WTFMove(mid)]() mutable {
```

Comment:
You don't need to move the sdp since the result of isolatedCopy is already a r-value reference.
```suggestion
    callOnMainThread([protectedThis = Ref(*this), this, sdp = candidateString.isolatedCopy(), sdpMLineIndex, descriptions = WTFMove(descriptions), mid = WTFMove(mid)]() mutable {
```